README-cfn.md

```diff
@@ -1,5 +1,7 @@
 # DBP Deployment on AWS EKS
 
+*Note:* When deploying to cloud environments like AWS and Azure you should consider using native database services from those providers rather than deploying Postgres within the Kubernetes cluster.
```

Comment:
My mistake, I forgot that the docs on this page already refer to RDS for the database so this note is not needed.